frontend/src/components/widgets/DataFrame/columns/ProgressColumn.test.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GridCellKind } from "@glideapps/glide-data-grid"
+import { RangeCellType } from "@glideapps/glide-data-grid-cells"
+
+import { BaseColumnProps, isErrorCell } from "./utils"
+import ProgressColumn, { ProgressColumnParams } from "./ProgressColumn"
+
+const PROGRESS_COLUMN_TEMPLATE = {
+  id: "1",
+  name: "progress_column",
+  title: "Progress column",
+  indexNumber: 0,
+  isEditable: false,
+  isHidden: false,
+  isIndex: false,
+  isStretched: false,
+  arrowType: {
+    // The arrow type of the underlying data is
+    // not used for anything inside the column.
+    pandas_type: "float64",
+    numpy_type: "float64",
+  },
+} as BaseColumnProps
+
+function getProgressColumn(
+  params?: ProgressColumnParams
+): ReturnType<typeof ProgressColumn> {
+  return ProgressColumn({
+    ...PROGRESS_COLUMN_TEMPLATE,
+    columnTypeOptions: params,
+  } as BaseColumnProps)
+}
+
+describe("ProgressColumn", () => {
+  // TODO(lukasmasuch): Implement test for step parameter
```

Comment:
Should these tests really be waited on to be implemented? Makes me slightly worried to not have tests for something that goes into develop.

Reply:
implementation looked fine for me. I'll give approval but please add some tests :)

Reply:
yep, I will :)

Reply:
Added a couple of additional tests :)